bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTEvaluationDetail.kt

```diff
@@ -0,0 +1,84 @@
+package io.bucketeer.sdk.android
+
+import io.bucketeer.sdk.android.internal.util.contains
+import org.json.JSONObject
+
+data class BKTEvaluationDetail<T>(
+  var id: String,
+  val featureId: String,
+  val featureVersion: Int,
+  val userId: String,
+  val variationId: String,
+  val variationName: String,
+  val variationValue: T,
+  val reason: Reason,
+) {
+  enum class Reason {
+    TARGET,
+    RULE,
+    DEFAULT,
+    CLIENT,
+    OFF_VARIATION,
+    PREREQUISITE,
+
+    ;
+
+    companion object {
+      fun from(value: String): Reason = entries.firstOrNull { it.name == value } ?: DEFAULT
```

Comment:
The `CLIENT` means that the SDK returned the default value to the app.
I think it's better if we return the `CLIENT` when we don't find the reason.

```suggestion
      fun from(value: String): Reason = entries.firstOrNull { it.name == value } ?: CLIENT
```